### ISSUE TITLE
strange error in laplace estimation of manual_priors example

### ISSUE THREAD

**Issue by tbmiller-astro** [pysersic/pysersic#13]
I saw there is a strange error in the example notebook where the variance from the posterior estimated with laplace method is zero and it spits out a "MAP is singular error"

I was able to replicate it! and there are a few solutions I've found:

1) Essentially any other random key I try produces a better fit with good uncertainties, so maybe it's just a really unlucky initialization? Still not sure what exactly the underlying cause is.

2) If instead of the normal 32 bit precision with jax, if I force it to 64 bit (see: https://jax.readthedocs.io/en/latest/notebooks/Common_Gotchas_in_JAX.html#double-64bit-precision) it also results in a good fit. I think the reason might be due to the matrix inversion that the Laplace method does (I think?) which sometimes behaves better with the higher precision. It could also be that it changes the initialization somehow to make it less unlucky.

Anyways we should at the very least change the random seed for that tutorial. I think better to stick to 32 bit if possible as it is faster than the 64 bit and we shouldn't need the extra precision in other contexts. Maybe also a better initialization scheme would help too?

**Comment by tbmiller-astro** [comment]
Ya there might be a way to catch it, if it happens more often would be good to figure out. I did some more testing and I think the issues might have also been with the learning rate. Decreasing it to 1e-2, was 1e-1, means it takes a little longer to converge but is much more consistent. We should just keep an eye out as we test on more things. Just committed a few changes that are better I think. 

Will keep this open for now to remind us.
